.pre-commit-config.yaml

```
@@ -0,0 +1,19 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.2.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-added-large-files
+- repo: https://github.com/psf/black
+  rev: 22.3.0
+  hooks:
+  - id: black
+- repo: https://github.com/asottile/pyupgrade
+  rev: v2.37.3
+  hooks:
+  - id: pyupgrade
+    args: [--py36-plus]
```

```suggestion
default_language_version:
  python: python3.9

repos:
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.2.0
    hooks:
    -   id: check-merge-conflict
    -   id: check-json
    -   id: check-yaml
    -   id: debug-statements
    -   id: end-of-file-fixer
        exclude: ^docs/.*$
    -   id: pretty-format-json
        args:
        - --autofix
    -   id: trailing-whitespace
        exclude: README.md
-   repo: https://github.com/asottile/pyupgrade
    rev: v2.32.1
    hooks:
    -   id: pyupgrade
-   repo: https://github.com/ambv/black
    rev: 22.3.0
    hooks:
    -   id: black
-   repo: https://github.com/PyCQA/flake8
    rev: 4.0.1
    hooks:
    -   id: flake8
```

Hey, great change!
We're already using `pre-commit` in the main repo and `sqlalchemy`. It would make sense to use the same file everywhere so we can just copy it over on changes

---

Flake8 has a separate config file in graphene